recipe/meta.yaml

```diff
@@ -53,6 +52,7 @@ outputs:
         - libpng
         # ffmpeg
         - numpy
+        # exclude 8.3.0 specifically due to pytorch/vision#4146, python-pillow/Pillow#5571
         - pillow >=4.1.1,<8.3
         - pytorch >=1.8.0={{ torch_proc_type }}*
```

Comment:
your recipe includes a pinning. I think this overrides conda forge's. Try removing this?

Reply:
I do'nt htink we pin gpu/cpu correctly. so maybe wait until we resolve things in

https://github.com/conda-forge/pytorch-cpu-feedstock/issues/40